src/components/form/Form.tsx

```diff
@@ -94,6 +101,9 @@ export function FormPage({ currentPageId }: { currentPageId: string | undefined
 
   return (
     <>
+      <Helmet>
+        <title>{`${getPageTitle(appName, langAsString(currentPageId), appOwner)}`}</title>
```

Comment:
As Ole Martin pointed out in slack, we should probably check that the currentPageId actually has a textResource associated with it before adding it. And if not, default to just `appName - appOwner` which is what we have today